.eslintrc

```diff
@@ -2,6 +2,7 @@
   "extends": "buildo",
   "plugins": ["flowtype"],
   "rules": {
+    "max-len": 0,
```

Comment:
Nice try. I'm not gonna approve this :grin:
I would set it at 80, but can we find a compromise at 120? Beyond that, it doesn't fit on GitHub.